### DISCUSSION TITLE
npx create-next-app fails every time i am using it

### DISCUSSION THREAD

**Discussion by atharva-joshi-searce** [vercel/next.js#48128] [Help]
### Summary

I am using npx create next app create a next app with next js, I am new to coding in JavaScript but I am a Python expert and I am unable to figure out what is the issue here I tried asking chat bots and Googling the error but there is no luck. Can anyone help me with this please this is critical as I need to develop an website with next js  within the next two months for project that I am working on. Can anyone please help me with this thank you. here Is the error that I get when I create next app

next dev   
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
warn  - Found lockfile missing swc dependencies, patching...
event - compiled client and server successfully in 580 ms (168 modules)
wait  - compiling /_error (client and server)...
event - compiled client and server successfully in 63 ms (169 modules)
Warning: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.
TypeError: Cannot read properties of null (reading 'useContext')

**Comment by atharva-joshi-searce** [comment]
Additional information here is that I am simply turning create next app I have not made any changes to the code whatsoever and running `next-dev` gives me this error , thanks !

**Reply by totti-rdz** [reply]
Duplicate issue, see https://github.com/vercel/next.js/issues/47048

Unfortunately I can not reproduce your error. Can you upload your project to github, so I can clone it and have a look?
